pyfar/dsp/dsp.py

```diff
@@ -223,3 +224,75 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     times -= times[0]
 
     return frequencies, times, spectrogram
+
+
+def deconv(measurement, excitation, **kwargs):
+    """Function to calculate the deconvolution of two signals.
+
```

Comment:
We could add information to make things really obvious for novices:

```
The transfer function :math:`H(\\omega)` is calculated by spectral
deconvolution (spectral division)

.. math::
   \\H(\\omega) = \\frac{Y(\\omega)}{X(\\omega)}, 

where :math:`X(\\omega)` is the excitation signal and :math:`Y(\\omega)`
the measured signal. Regulated inversion is used to avoid numerical issues
in calculating :math:`\\hat{X(\\omega)} = 1/X(\\omega)` for small values of
:math:`X(\\omega)` (see :py:func:`pyfar.dsp.regulated_spectrum_inversion`).
The transfer function is thus calculated as

.. mat::
    \\H(\\omega) = Y(\\omega)\\hat{X(\\omega)}.

For more information, refer to [#]_
```

and add below

```
References
-----------
.. [#] S. Mueller and P. Masserani "Transfer function measurement with sweeps. Directors cut.
       J. Audio Eng. Soc. 49(6):443-471, (2001, June).
```

Reply:
Thank you for that suggestion, I added this with my latest commit.